requirements.txt

```diff
@@ -1,3 +1,4 @@
+wheel 
```

Comment:
I removed wheel because further investigation The bdist_wheel command error I saw without the wheel package does not indicate the installation failed https://stackoverflow.com/questions/59104686/wheel-installation-from-within-requirements-txt.